tests/ui/macros/remove-repetition-issue-139480.rs

```diff
@@ -0,0 +1,28 @@
+macro_rules! ciallo {
+    ($($v: vis)? $name: ident) => {
+    //~^ error: repetition matches empty token tree
+    };
+}
+
+macro_rules! meow {
+    ($name: ident $($v: vis)?) => {
+    //~^ error: repetition matches empty token tree
+    };
+}
+
+macro_rules! gbc {
+    ($name: ident $/*
+        It's MyGO!!!!!
```

Comment:
?

Reply:
Can you remove the unnecessary comments? It's a bit distracting

Reply:
```suggestion
        this comment gets removed by the suggestion
```
Sorry about that. Would this version be better in your opinion?

Reply:
That's much easier to understand, yes